repositories/mysql.go

```diff
@@ -26,6 +27,9 @@ func NewDatabaseConnection(dsn string) (*gorm.DB, error) {
 
 	return gorm.Open(mysql.Open(dsn), &gorm.Config{
 		Logger: logger.Default.LogMode(logger.Silent),
+		NamingStrategy: schema.NamingStrategy{
+			SingularTable: true,
```

Comment:
Where is this change for? If you needed this, then I am very curious why, since all table names are configured manually. Did you create a new one?

Reply:
Without it, I get below error.

```
twomes-backoffice-api-local-web  | 2023/06/02 08:18:59 /go/src/twomes-api-server/repositories/device.go:109 Error 1146 (42S02): Table 'twomes.measurements' doesn't exist
twomes-backoffice-api-local-web  | [0.381ms] [rows:0] SELECT `measurements`.`id`,`measurements`.`upload_id`,`measurements`.`property_id`,`measurements`.`time`,`measurements`.`value` FROM `measurements` JOIN upload ON measurement.upload_id = upload.id JOIN device ON upload.device_id = device.id WHERE device.id = 5 AND measurement.time >= '2023-05-09T12:33:36.324Z'
```

Reply:
I will have to look into that, but thank you!